internal/output/utils.go

```diff
@@ -75,13 +87,126 @@ func MakeTable(rows []Row) ([]string, []Row) {
 				v.Name = h
 			}
 			row[i] = v
+			sv := fmt.Sprint(v.Value)
+			if len(sv) > width[i] {
+				width[i] = len(sv)
+			}
+			if !alignSet {
+				align[i] = alignmentForType(v.Type)
+			}
 		}
+		alignSet = true
 		rows[ri] = row
 	}
 	// remove this. prefix from header cells
 	thisPfx := fmt.Sprintf("%s.", NameValue)
+	columns := make(table.Row, len(header))
 	for i, h := range header {
-		header[i] = strings.TrimPrefix(h, thisPfx)
+		columns[i] = table.Column{
+			Header: strings.TrimPrefix(h, thisPfx),
+			Align:  align[i] * width[i],
+		}
+	}
+	return columns, rows
+}
+
+func makeTableHeaderFromRow(row Row, maxWidth int) table.Row {
+	dw := defaultWidthForFlexibleColumns(row, maxWidth)
+	// the max width for a column is 1000, just an arbitrary decision --YT
```

Comment:
We have used max width only for header. Maybe we can set this value lower and also we have to use for both header and also column the same value.

Reply:
The columns use the same width/alignment with the header. This code is used only when the table is in "streaming mode" (the max width of columns not found) as a safety measure.